iroh/src/magicsock/relay_actor.rs

```diff
@@ -159,6 +162,20 @@ struct ActiveRelayActor {
     /// Token indicating the [`ActiveRelayActor`] should stop.
     stop_token: CancellationToken,
     metrics: Arc<MagicsockMetrics>,
+    /// Received relay packets that could not yet be forwarded to the magicsocket.
+    pending_received: Option<PendingRecv>,
```

Comment:
Shouldn't this live on `ConnectedRelayState`?

Reply:
I moved this into a new struct together with the queue senders in the latest commit, but still kept it on the `ActiveRelayActor`. My reasoning: If the connection to a relay breaks, but we still have pending received items queued, we can still try pushing those to the magicsock.